settings.php

```diff
@@ -368,12 +387,12 @@
 // fast_404();
 
 /**
- * External access proxy settings:
+ * External access proxy settings.
  *
- * If your site must access the Internet via a web proxy then you can enter
- * the proxy settings here. Currently only basic authentication is supported
- * by using the username and password variables. The proxy_user_agent variable
- * can be set to NULL for proxies that require no User-Agent header or to a
+ * If your site must access the Internet via a web proxy then you can enter the
+ * proxy settings here. Currently only basic authentication is supported  by
```

Comment:
Remove extra space

```suggestion
 * proxy settings here. Currently only basic authentication is supported by
```

Reply:
Thanks @yorkshire-pudding 🙏🏼 ...and sorry for the late reply. Committed 👍🏼